languages/csharp/exercises/concept/properties/.docs/introduction.md

```diff
@@ -0,0 +1,31 @@
+A property in C# is a member of a class that provides access to data within that class.
+Callers can set or retrieve (get) the data.  Properties can be either auto-implemented or
+have a backing field.  
+
+When setting a property the input value can be validated, formatted
+or otherwise manipulated and in fact any programmatic operation accessible to code in the
+class can be executed.  
+
+Similarly when retrieving a property data can be calculated or formatted and again
+any programmatic operation available to the class can be executed.
+
+Properties have access modifiers (`public`, `private` etc.) in the same way as other
+class members but the set accessor may have an access level independent of the retrieve (get)
+accessor and vice versa or either accessor may be dispensed with completely.
```

Comment:
I was wondering if perhaps if would be helpful to stress that properties can have one accessor if they want. As a non-native English speaker, I found the "dispensed with completely" not immediately obvious (could be me of course). Perhaps something like this?

```suggestion
accessor and vice versa. A property doesn't have to have both accessors, it can have just one (either get or set).
```